### PR TITLE
opam 2.0.2

### PR DESCRIPTION
**Pull request by rjbou**

### CODE REVIEW

File: opam-2-0-2.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+date: "2018-12-11"
+--BODY--
+
+We are pleased to announce the release of [opam 2.0.2](https://github.com/ocaml/opam/releases/tag/2.0.2).
+
+This new version contains mainly [backported fixes](https://github.com/ocaml/opam/pull/3669):
+* Doc:
+  * update man page
+  * add message for deprecated options
+  * reinsert removed ones
```
Comment (avsm) [inline opam-2-0-2.md]: Just a request for a slightly more user facing change log - what does “reinsert removed ones” mean? :)

File: opam-2-0-2.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  * deprecate `no-aspcud`
+* Pin:
+  * upgrade pin depends on pinning
```
Comment (avsm) [inline opam-2-0-2.md]: Not sure what this one is referring to — is this the issue @hannesm reported about pin-depends?

File: opam-2-0-2.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  * include descr & url files on pinning 1.2 opam files
+* Sandbox:
+  * handle symlinks in bwrap
```
Comment (avsm) [inline opam-2-0-2.md]:
```suggestion
  * handle symlinks in bubblewrap for system directories such as `/bin` or `/lib` ([#3661](https://github.com/ocaml/opam/pull/3661)).  Fixes sandboxing on some distributions such as CentOS 7 and Arch Linux.
```

File: opam-2-0-2.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  * allow use of internal sockets on Mac OS
```
Comment (avsm) [inline opam-2-0-2.md]:
```suggestion
  * allow use of unix domain sockets on macOS ([#3659](https://github.com/ocaml/opam/issues/3659))
```

File: opam-2-0-2.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  * change one-line conditional to if statement which was incompatible with set -e
+  * make /var readonly instead of empty and rw
+* Path: resolve default opam root path
+* System: suffix .out for read_command_output stdout files
+* Locked: check consistency with opam file when reading lock file to suggest regeneration message
+* Show: remove pin depends messages
+* Cudf: Fix closure computation in the presence of cycles
```
Comment (avsm) [inline opam-2-0-2.md]: what is the implication of this fix?

File: opam-2-0-2.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+* List: Fix some cases of listing coinstallable packages
+* Format upgrade: extract archived source files of version-pinned packages
+* Core: add is_archive in OpamSystem and OpamFilename
+* Init: don't fail if empty compiler given
+* Lint: fix light_uninstall flag for error 52
+* Build: partial port to dune
+* Update cold compiler to 4.07.1
+
+As **sandbox scripts** have been updated, don't forget to run `opam init --reinit -ni` to update yours.
```
Comment (avsm) [inline opam-2-0-2.md]: This should probably be at the top and in bold :)

File: opam-2-0-2.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+Installation instructions (unchanged):
+
+1. From binaries: run
+
+    ```
+    sh <(curl -sL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh)
+    ```
+
+    or download manually from [the Github "Releases" page](https://github.com/ocaml/opam/releases/tag/2.0.2) to your PATH. In this case, don't forget to run `opam init --reinit -ni` to enable sandboxing if you had version 2.0.0~rc manually installed or to update you sandbox script.
```
Comment (avsm) [inline opam-2-0-2.md]:
```suggestion
    or download manually from [the Github "Releases" page](https://github.com/ocaml/opam/releases/tag/2.0.2) to your PATH. In this case, don't forget to run `opam init --reinit -ni` to enable sandboxing if you had version 2.0.0~rc manually installed or to update your sandbox script.
```

File: opam-2-0-2.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+    ```
+    opam update; opam install opam-devel
+    ```
+
+   (then copy the opam binary to your PATH as explained, and don't forget to run `opam init --reinit -ni` to enable sandboxing if you had version 2.0.0~rc manually installed or to update you sandbox script)
+
+3. From source, manually: see the instructions in the [README](https://github.com/ocaml/opam/tree/2.0.2#compiling-this-repo).
+
+We hope you enjoy this new major version, and remain open to [bug reports](https://github.com/ocaml/opam/issues) and [suggestions](https://github.com/ocaml/opam/issues).
```
Comment (avsm) [inline opam-2-0-2.md]:
```suggestion
We hope you enjoy this new minor version, and remain open to [bug reports](https://github.com/ocaml/opam/issues) and [suggestions](https://github.com/ocaml/opam/issues).
```

**Comment by rjbou** [comment]
@avsm Thanks!

File: opam-2-0-2.md (hunk 2 above)
Comment (rjbou) [reply opam-2-0-2.md]: yes! updated

File: opam-2-0-2.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+authors: [
+  "Raja Boujbel - OCamlPro" {"mailto:raja.boujbel(à)ocamlpro.com"}
+  "Louis Gesbert - OCamlPro" {"mailto:louis.gesbert(à)ocamlpro.com"}
+]
+date: "2018-12-12"
+--BODY--
+
+We are pleased to announce the release of [opam 2.0.2](https://github.com/ocaml/opam/releases/tag/2.0.2).
+
+As **sandbox scripts** have been updated, don't forget to run `opam init --reinit -ni` to update yours.
```
Comment (avsm) [inline opam-2-0-2.md]: do we have an motd facility in opam2? we could update the motd on the main opam-repository to show this message if so

File: opam-2-0-2.md [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+
+This new version contains mainly [backported fixes](https://github.com/ocaml/opam/pull/3669):
+* Doc:
+  * update man page
+  * add message for deprecated options
+  * reinsert removed ones to print a deprecated message instead of fail (e.g. `--alias-of`)
+  * deprecate `no-aspcud`
+* Pin:
+  * on pinning, rebuild updated pin depends packages
```
Comment (avsm) [inline opam-2-0-2.md]:
```suggestion
  * on pinning, rebuild updated `pin-depends` packages reliably
```

File: opam-2-0-2.md (hunk 9 above)
Comment (rjbou) [reply opam-2-0-2.md]: We can update the package in the repo, but it would be available only to user that update via the package `opam-devel`.

**Pull request merged**